.github/workflows/containerize.yml

```diff
@@ -0,0 +1,43 @@
+name: Containerize
+on:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  containerize:
+    runs-on: ubuntu-22.04
+    name: Containerize
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
When `fetch-depth` != 1 probably you want `ref: ${{ github.event.pull_request.head.sha }}` too, otherwise you'll get a merge commit that does not actually exist on the repo.

https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit

Reply:
Thank you for the review. Fixed to use `fetch-depth` == 1